nextcord/http.py

```diff
@@ -879,12 +879,12 @@ def ban(
         self,
         user_id: Snowflake,
         guild_id: Snowflake,
-        delete_message_days: int = 1,
+        delete_message_seconds: int = 0,
```

Comment:
Maybe this should be the number of seconds in a day in case it is called directly. If it isn't ever called without it, there wouldn't be a need for a default at all.
```suggestion
        delete_message_seconds: int = 86400,
```